packages/reports/src/Stores/Modules/Profit/Helpers/format-response.ts

```diff
@@ -0,0 +1,37 @@
+import { formatMoney, toMoment, getSymbolDisplayName, getMarketInformation } from '@deriv/shared';
+import { ActiveSymbols, ProfitTableResponse } from '@deriv/api-types';
+
+export type TTransaction = NonNullable<NonNullable<ProfitTableResponse['profit_table']>['transactions']>[number];
+
+export const formatProfitTableTransactions = (
+    transaction: TTransaction,
+    currency: string,
+    active_symbols: ActiveSymbols = []
+) => {
+    const format_string = 'DD MMM YYYY HH:mm:ss';
+    const purchase_time = transaction.purchase_time && `${toMoment(+transaction.purchase_time).format(format_string)}`;
+    const purchase_time_unix = transaction.purchase_time;
+    const sell_time = transaction.sell_time && `${toMoment(+transaction.sell_time).format(format_string)}`;
+    const payout = transaction.payout ?? NaN;
```

Comment:
Why do we have to explicitly assign it as NaN?

Reply:
because in profit loss we need to subtract sell price and buy price. It doesnt work if its undefined

Reply:
yes, it's a correct replacement:
`parseFloat()` was used here to ensure that if the value is `undefined` or `null`, it's converted into `NaN`. For example, `transaction.payout` can come as a `number`, `null` or `undefined` from API. 
But parseFloat() in TypeScript accepts only a `string`, that's why we fallback to `NaN` to keep the existing functionality.